src/control/server/harness.go

```diff
@@ -88,20 +88,28 @@ func (h *IOServerHarness) AddInstance(srv *IOServerInstance) error {
 
 // GetManagementInstance returns a managed IO Server instance
 // to be used as a management target.
-func (h *IOServerHarness) GetManagementInstance() (*IOServerInstance, error) {
+func (h *IOServerHarness) GetManagementInstance() (*IOServerInstance, bool, error) {
```

Comment:
This is a little awkward, because it means that anything that needs to be aware of the instance's MS replica status has to call getMgmtInfo() again directly or else carry around the boolean value returned from this function. Plus, that function isn't really integrated very well with all of the new code -- all of the MS stuff was moved mostly unmodified without much effort to integrate it until it is ready to use in closer to its final form.

The information we need should be in the superblock which is read or created in harness.CreateSuperblocks().

It seems like we should be able to add a convenience method to IOServerInstance, e.g. 
```go
func (srv *IOServerInstance) IsReplica() bool {
        if !srv.hasSuperblock() {
                  return false
        }
        return srv.getSuperblock().MS
}
```

Reply:
will make change, did feel a little awkward modifying that function